Sources/Parley/Views/ParleyTextView/ParleyTextView.swift

```diff
@@ -5,19 +5,10 @@ final class ParleyTextView: UITextView {
 
     var appearance = ParleyTextViewAppearance() {
         didSet {
-            linkFont = appearance.linkFont
-            regularFont = appearance.regularFont
-            italicFont = appearance.italicFont
-            boldFont = appearance.boldFont
             updateAttributedText()
         }
     }
 
-    @ParleyScaledFont(textStyle: .body) var linkFont = .systemFont(ofSize: 14)
-    @ParleyScaledFont(textStyle: .body) var regularFont = .systemFont(ofSize: 14)
-    @ParleyScaledFont(textStyle: .body) var italicFont = .italicSystemFont(ofSize: 14)
-    @ParleyScaledFont(textStyle: .body) var boldFont = .boldSystemFont(ofSize: 14)
-
```

Comment:
❌ Regression:

These seemingly 'duplicated' fonts were needed to make the font scaling working while in-app. 

This change (removing these and putting them at `ParleyTextViewAppearance` breaks this realtime updating of the prefered size in the ParleyTextView.

Can be reproduced by opening the sample in the simulator, then press `CMD`+`Option`+`+` while the chat is open -- ParleyTextView won't automatically update it's font size anymore when removing this here.